docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -55,7 +55,7 @@ that serves up a very simple web page, conveyed via a Kubernetes
 ConfigMap that is mounted as a volume for the httpd pod.
 
 ```shell
-kubectl apply -f - <<EOF
+kubectl --kubeconfig $wmw_c_space_config apply -f - <<EOF
```

Comment:
"kubeconfig" rather than "config" in that variable name would make its meaning more obvious.